vivisect/tests/testvivisect.py

```diff
@@ -494,8 +503,8 @@ def test_vaset_populate(self):
                             ('program_invocation_name', 0x8061300),
                             ('program_invocation_short_name', 0x80612f0),
                             ('__gmon_start__', 0x8060ffc)],
-            'PointersFromFile': [(134614800, 134519888, 'vdir', 'fini_function_0'),
-                                 (134614796, 134519936, 'vdir', 'init_function_0'),
+            'PointersFromFile': [(0x8060f0c, 0x8049c80, 'vdir', '__frame_dummy_init_array_entry'),
+                                 (0x8060f10, 0x8049c50, 'vdir', '__do_global_dtors_aux_fini_array_entry'),
```

Comment:
prefer to assert the presence of the symbol name provided by the ELF file

Reply:
i'm a fan.  
does the new code handle this naming differently?

Reply:
no, wait.  nm.  it seems we name them one way first, then apply different names later

```
2024-08-12 20:44:41,195:DEBUG:vivisect.parsers.elf: symbol val: 0x8049c50       type: 'Symbol is a code object' bind: 'Local symbol'     name: '__do_global_dtors_aux'
2024-08-12 20:44:41,195:DEBUG:vivisect.base: replacing 0x8049c50: 'vdir.fini_function_0' -> 'vdir.__do_global_dtors_aux'
2024-08-12 20:44:41,195:DEBUG:vivisect.parsers.elf: symbol val: 0x806131c       type: 'Symbol is a data object' bind: 'Local symbol'     name: 'completed.7282'
2024-08-12 20:44:41,195:DEBUG:vivisect.parsers.elf: symbol val: 0x8060f10       type: 'Symbol is a data object' bind: 'Local symbol'     name: '__do_global_dtors_aux_fini_array_entry'
2024-08-12 20:44:41,196:DEBUG:vivisect.parsers.elf: symbol val: 0x8049c80       type: 'Symbol is a code object' bind: 'Local symbol'     name: 'frame_dummy'
2024-08-12 20:44:41,196:DEBUG:vivisect.base: replacing 0x8049c80: 'vdir.init_function_0' -> 'vdir.frame_dummy'
2024-08-12 20:44:41,196:DEBUG:vivisect.parsers.elf: symbol val: 0x8060f0c       type: 'Symbol is a data object' bind: 'Local symbol'     name: '__frame_dummy_init_array_entry'
```

so it looks like we're naming the location of the symbol and the location the symbol points to:

```In [1]: vw.getName(0x8060f10)
Out[1]: 'vdir.__init_array_end'

In [2]: vw.getName(0x8060f0c)
Out[2]: 'vdir.__init_array_start'

In [3]: vw.getName(0x8049c50)
Out[3]: 'vdir.__do_global_dtors_aux'

In [4]: vw.getName(0x8049c80)
Out[4]: 'vdir.frame_dummy'
```

and apparently the unittests don't care, that's just there for troubleshooting.

so i'm ok with the change.  but these names are slightly different than what you have.